### ISSUE TITLE
Go.property resetting to default after changing source-go

### ISSUE THREAD

**Issue by MaratGilyazov** [defold/editor2-issues#2554]
go.properties reset to def when I edit the source game-object.

So, I have a "Can" game object with sprite, collision-object and script on it, and in script there are some custom properties. I have levels as collections with Cans set in different states all over the levels. And when I edit collision-object properties of the Can GO - all custom properties reset to default.

<img width="403" alt="Screenshot+2019-04-11+10 29 31" src="https://user-images.githubusercontent.com/15916299/56086089-9cf15600-5e4f-11e9-821b-403f91be6c21.png">
<img width="490" alt="Screenshot+2019-04-11+10 30 51" src="https://user-images.githubusercontent.com/15916299/56086090-9cf15600-5e4f-11e9-8227-a3b635e61d48.png">
<img width="977" alt="Screenshot+2019-04-11+10 59 06" src="https://user-images.githubusercontent.com/15916299/56086091-9d89ec80-5e4f-11e9-889e-b97545f46fa8.png">

**Comment by matgis-king** [comment]
Hi! I was not able to reproduce the issue from the description. Would you be willing to share the project with `mats.gisselson@king.com` so we can take a closer look?

**Comment by MaratGilyazov** [comment]
Hi, @matgis-king, I just tried and also was not able to repro by the description I gave. I've rechecked the old commit when I've broke everything and found out that "Can" was modified in previous commit, and in next commit where all levels lost custom properties for Cans, I've moved one of the GameObjects used in every level-collection to other directory, what caused modification of all levels. 

But I've tried to repro and yes, all level-collections are modified, but no custom property was lost. Maybe something like this was fixed recently?
